### PR TITLE
CONSOLE-4127: replace deprecated components in UtilizationDurationDro…

### PR DESCRIPTION
**Pull request by rhamilto**
…pdown

This is 1 of 2 PRs to address https://issues.redhat.com/browse/CONSOLE-4127

After

https://github.com/user-attachments/assets/3d37b1bb-b49b-4223-96a4-143af14365b8

### CODE REVIEW

**Comment by rhamilto** [comment]
Tech debt so manually adding labels:
/label px-approved
/label docs-approved

/assign @yapei
/assign @sg00dwin

File: frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationDurationDropdown.tsx [hunk 1]
```diff
@@ -25,27 +21,36 @@ export const UtilizationDurationDropdown: React.FC<UtilizationDurationDropdownPr
...
+          </MenuToggle>
+        )}
         onSelect={onSelect}
-        selections={selectedKey}
+        selected={selectedKey}
+        onOpenChange={(open) => setIsOpen(open)}
         isOpen={isOpen}
         isPlain
       >
         {Object.keys(items).map((key) => (
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/components/dashboard/utilization-card/UtilizationDurationDropdown.tsx:48]: Missing `<SelectList>`

**Review by sg00dwin — APPROVED** [review]
/lgtm

**Pull request merged**